compiler/passes/codegen.cpp

```diff
@@ -409,6 +409,61 @@ static void genFilenameTable() {
   genGlobalInt32(sizeName, gFilenameLookup.size());
 }
 
+//
+// This adds the Chapel symbol table to the config file
+// Our symbol table is formed by two 1-D arrays with 2 elements
+// per entry:
+//
+// chpl_funSymTable     = cname, Chapel name
+// chpl_filenumSymTable = Chapel file name index, Chapel line number
+//
+static void genUnwindSymbolTable(){
+  GenInfo *info = gGenInfo;
+
+  //TODO should handle LLVM
```

Comment:
I don't expect you to write the LLVM version - so this could just say "TODO: Could have a native LLVM version, instead of relying on C to LLVM"